storage/rethinkdb/bootstrap.go

```diff
@@ -16,7 +17,6 @@ func makeDB(session *gorethink.Session, name string) error {
 
 		return err
 	}
-	// Note: Wait() not supported by RethinkDB 2.3+
```

Comment:
Note that `Wait()` is supported in Rethink 2.3, just that it expects a DB or Table to be specified: https://rethinkdb.com/api/javascript/wait/